src/vaillant_netatmo_api/thermostat.py

```diff
@@ -353,6 +414,11 @@ def __init__(
         system_mode: str | None = None,
         setpoint_hwb: dict = {},
         modules: list[dict] = [],
+        dhw_enabled: bool | None = None,
```

Comment:
I suggest changing the default value to `False` not `None`, as it's always returned from the API.

Making it "trilean" (ie. true, false, null 😄) just complicates the callers code unnecessary.